packages/web/src/scss/components/FileUploader/README.md

```diff
@@ -11,27 +11,13 @@ FileUploader is a composition of a few subcomponents:
   - [FileUploaderList](#fileuploaderlist)
     - [FileUploaderAttachment](#fileuploaderattachment)
 
-## JavaScript Plugin (TO BE IMPLEMENTED)
-
-For full functionality, you need to provide Spirit JavaScript, which will handle
-the interactions with FileUploader.
-
-```html
-<script src="node_modules/@lmc-eu/spirit-web/js/cjs/spirit-web.min.js" async></script>
-```
-
-Please consult the [main README][web-readme] for how to include JavaScript
-plugins.
-
-Or, feel free to write the controlling script yourself.
-
 ## FileUploader
 
 This is the main wrapper for the whole composition. It provides proper spacing
 for its subcomponents:
 
 ```html
-<div class="FileUploader">
+<div class="FileUploader" data-toggle="fileUploader">
```

Comment:
no spirit infix?

Reply:
This will be solved more than once for all components due to the fact that we have `data-toggle` as the main initializer of JS scripts